multisig-factory/src/lib.rs

```diff
@@ -32,19 +27,20 @@ impl MultisigFactory {
         members: Vec<MultisigMember>,
         num_confirmations: u64,
     ) -> Promise {
-        let account_id = format!("{}.{}", name, env::current_account_id());
+        let account_id =
+            AccountId::try_from(format!("{}.{}", name, env::current_account_id())).unwrap();
         Promise::new(account_id)
             .create_account()
             .deploy_contract(CODE.to_vec())
             .transfer(env::attached_deposit())
             .function_call(
```

Comment:
Let's use [function_call_weight](https://docs.rs/near-sdk/latest/near_sdk/struct.Promise.html#method.function_call_weight) with GasWeight(1) to pass all the unused gas down to the new function